README.md

```diff
@@ -1 +1,184 @@
-# Blog-App
\ No newline at end of file
+# README
+<a name="readme-top"></a>
+
+This README would normally document whatever steps are necessary to get the
+application up and running.
+<div align="center">
+
+Things you may want to cover:
+  <h1><b>Blog App</b></h1>
+
+* Ruby version
+</div>
+
+* System dependencies
+<!-- TABLE OF CONTENTS -->
+
+* Configuration
```

Comment:
- [Optional] I think all these text are unnecessary where you've placed; I suggest you remove them to make your README to be professional. 👍